### ISSUE TITLE
Issue with /docs/tutorials/kubernetes-basics/deploy-interactive/

### ISSUE THREAD

**Issue by joshattr** [kubernetes/website#1529]
**This is a...** 

- [ ] Feature Request
- [x] Bug Report

**Problem:**
When you click some code to run it in the terminal (e.g. `kubectl deployments ↩`) it _always_ is pasted into the most recent terminal you used to run a similar expression (not any expression, but one you click on, with a `↩` at the end, let's call them "documentation expressions").

This causes problems when the last code you ran by clicking a documentation expression stays in the foreground, like `kubectl proxy ↩`, because the next documentation expression you click will never be run, even if you make more terminals. It always switches you back and pastes the code in below the command that's running in the foreground.

**Proposed Solution:**
Use the currently visible terminal to run expressions from the documentation.

**Page to Update:**
I experienced this here: http://kubernetes.io/docs/tutorials/kubernetes-basics/deploy-interactive/, but would assume it's common to all the interactive docs.

**Comment by joshattr** [comment]
Hmm... I thought typing `kubectl proxy` in a different web terminal would allow me to continue clicking the documentation expressions... but it looks like they try to run in the `kubectl proxy` web terminal no matter what. Maybe it's using the oldest terminal or something?

**Comment by BenHall** [comment]
Thank you for the feedback. We're working on getting this improved.

**Comment by joshattr** [comment]
:beers: thanks!

**Comment by vhosakot** [comment]
I also saw this issue today and observed that `kubectl proxy` blocks and runs in the foreground and we cannot continue with the tutorial.  Hence, I just used `&` and ran it in the background, and then, I was able to run all the subsequent commands successfully.

`kubectl proxy &`

Let me know if can I submit a PR to add `&` for the command `kubectl proxy` in [https://kubernetes.io/docs/tutorials/kubernetes-basics/deploy-interactive/](url).
